CONTRIBUTORS.md

```diff
@@ -1,8 +1,10 @@
-Cerner Corporation
+# Cerner Corporation
+
 - Joshua Kuestersteffen [@jkuester]
 - Matej Voboril [@TobiTenno]
-- Matthew Beers [@beersonthewall]
+- Nikita Prabhakar [@nikitaprabhakar]
 
 [@jkuester]: https://github.com/jkuester
 [@TobiTenno]: https://github.com/TobiTenno
-[@beersonthewall]: https://github.com/beersonthewall
\ No newline at end of file
+[@beersonthewall]: https://github.com/beersonthewall
```

Comment:
do you wanna move matthew's part to an "outside contributors" section?

Reply:
Oh yeah, good call.  Did not mean to remove him from the file completely.